lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -138,6 +139,11 @@ export abstract class BaseManagedIdentitySource {
 
         const networkRequestOptions: NetworkRequestOptions = { headers };
 
+        if (managedIdentityRequest.claims) {
+            networkRequest.bodyParameters[AADServerParamKeys.CLAIMS] =
```

Comment:
Not sure if this is right. Are we sending the claims to the endpoint? Maybe I am missing something.

Reply:
this design has been changing, and is still not final. Initially the ask was just to bypass cache and get a new token from the IMDS endpoint when claims was passed. So this is good as is. But there is a new change coming and more details to follow, but for now let's just skip the cache and get a new token.